src/components/Login/index.jsx

```diff
@@ -1,119 +1,137 @@
+  handleLoginSubmit,
+  handleInputChange,
+  handleAlertClose,
+}) => (
+  <div className="login-container">
+    <Row>
+      <Column width={10} offset={2}>
+        <div className="login-form-container">
+          <div className="login-logo">
+            <img
+              src="https://uiux.s3.amazonaws.com/toggleable-logos/header-logo.svg"
+              alt="sg-logo"
+            />
+          </div>
+          <Row width={8} offset={2}>
+            {!isAuthenticating &&
+              isAuthenticationError &&
+              isInvalidCredentials && (
+                <InvalidCredentialsAlert
+                  data-test="invalid-credentials-alert"
+                  handleAlertClose={handleAlertClose}
+                />
+              )}
+            {!isAuthenticating &&
+              isAuthenticationError &&
+              isNetworkError && (
+                <NetworkErrorAlert
+                  data-test="network-error-alert"
+                  handleAlertClose={handleAlertClose}
+                />
+              )}
+            {!isAuthenticating &&
+              isAuthenticationError &&
+              isInvalidInput && (
+                <InvalidInputAlert
+                  data-test="invalid-input-alert"
+                  handleAlertClose={handleAlertClose}
                 />
-              </div>
-              <div className="input-text-wrap">
-                <input
+              )}
+          </Row>
+          <div className="login-form-body">
+            <Row>
+              <Column width={6} offset={4}>
+                <div className="input-text-wrap">
+                  <TextInput
+                    data-test="username-field"
+                    onChange={e => handleInputChange(e, "username")}
+                    value={username}
+                    type="text"
+                    label="Username"
+                  />
+                </div>
+              </Column>
+            </Row>
+            <Row>
+              <Column width={6} offset={4}>
+                <TextInput
+                  onChange={e => handleInputChange(e, "password")}
+                  data-test="password-field"
+                  value={password}
                   type="password"
-                  placeholder="Password"
-                  onChange={e => this.updateField(e, "password")}
+                  label="Password"
                 />
-              </div>
-            </div>
-            <button
-              type="submit"
-              className="btn btn-primary"
-              onClick={this.handleLogin}
-            >
-              Login
-            </button>
+              </Column>
+            </Row>
+            <Row>
+              <Column width={6} offset={4}>
+                <div className="login-button-container">
+                  {!isAuthenticating && (
+                    <LoginButton
+                      data-test="login-button"
+                      handleLoginSubmit={handleLoginSubmit}
```

Comment:
I just noticed now that we're handling the login through just a button onClick handler. When it comes to inputs and data it's often more semantic (meaning it shows the intention more for what it should be doing) and clearer to wrap the textinputs and login button/loading button inside a form element with an onSubmit handler and the handleLoginSubmit would go into there like this:
```js
<form onSubmit={handleLoginSubmit}>
...TextInputs...
...Login/Loading button (they won't need the handleLoginSubmit function passed down anymore since the form has an onSubmit with the proper function but our ui-components require a prop of isSubmit i believe or something like that for it to add type="submit" to the button DOM element for it to trigger the form's onSubmit handler...
</form>
```